src/Tests/UnitTest/TestComponentCSharp_Tests.cs

```diff
@@ -3191,6 +3191,42 @@ public void TestActivationFactoriesFromMultipleContexts()
 
             Assert.Null(exception);
         }
+
+        class CustomGuidHelper : IGuidHelperStatics

```

Comment:
How were we able to implement this interface, it looks to be projected as an internal interface?

Reply:
Hmm that's a good point actually. It compiled as I ran the test locally. Let me see why.